app/assets/stylesheets/location-autocomplete.min.scss

```diff
@@ -17,3 +17,10 @@
 .govuk-form-group--error .autocomplete__wrapper input {
     border: 2px solid $govuk-error-colour;
 }
+
+// Make the autocomplete dropdown size correct (width-two-thirds) for all devices except mobile
+@media (min-width:40.0625em) {
```

Comment:
@jassalrichy we can use the media queries defined in govuk-frontend:
```
// Every thing up to desktop excluding mobile
@include govuk-media-query($from: mobile) {
}
```
https://github.com/alphagov/govuk-frontend/blob/4b5ed0ada3b64722ae2074dfa0db5d4d3a45cb79/src/govuk/helpers/_media-queries.scss